models/Document/Editable.php

```diff
@@ -330,6 +342,28 @@ public function addConfig(string $name, mixed $value): static
         return $this;
     }
 
+    public function getLabel(): ?string
+    {
+        return $this->label ?? null;
```

Comment:
Null coalescing is unnecessary, since the properties have a `null` value by default.

```suggestion
        return $this->label;
```